third_party/terraform/resources/resource_bigquery_table.go.erb

```diff
@@ -302,6 +302,55 @@ func resourceBigQueryTable() *schema.Resource {
 				},
 			},
 
+			<% unless version == 'ga' -%>
+			// RangePartitioning: [Optional] If specified, configures range-based
+			// partitioning for this table.
+			"range_partitioning": &schema.Schema{
+				Type: schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						// Field: [Required] The field used to determine how to create a range-based
+						// partition.
+						"field": {
+							Type:     schema.TypeString,
+							Required: true,
+							ForceNew: true,
+						},
+
+						// Range: [Required] Information required to partition based on ranges.
+						"range": &schema.Schema{
+							Type: schema.TypeMap,
```

Comment:
Sorry, I missed this the first time around - I belive this should be Type: schema.TypeList,